### ISSUE TITLE
When to expect tutorial for anmm model and listwise loss?

### ISSUE THREAD

**Issue by ctrado18** [NTMC-Community/MatchZoo#782]
Hi,

I just want to know If you can expect a tutorial in wikiqa for the anmm model like for the others to? Otherwise, It is a bit too hacky to implement this by myself.

Also, is it right now possible to use a listwise loss like the pairwise loss with any model?
I want to train some of the model in the tutorial like arci for a listwise loss. Is this possible right now?
I want to mark relevant, and also partial relevant as also non relevant documents with this loss.

**Comment by matthew-z** [comment]
IIRC, Rank cross entropy loss is a kind of listwise loss

**Comment by uduse** [comment]
I don't think it will be added anytime soon.

**Comment by yangliuy** [comment]
Hi @ctrado18, thank you for your message!

You can refer to the performance of aNMM model on WikiQA data here https://github.com/NTMC-Community/MatchZoo/blob/1.0/README.md . The implementation of aNMM based on Keras is here https://github.com/NTMC-Community/MatchZoo/blob/1.0/matchzoo/models/anmm.py. Hope this helps.
